docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,278 @@ task.download_outputs()
+machine_group.start()
+
+import inductiva
+
+# Set simulation input directory
+input_dir = "/path/to/highLiftConfiguration"
+
+# Read the simulation commands
+with open(os.path.join(input_dir,'input.txt'), 'r') as file:
+    commands = [line.strip() for line in file]
+
+# Initialize the Simulator
+openfoam = inductiva.simulators.OpenFOAM(distribution="esi")
+
+# Run simulation
+task = openfoam.run(
+    input_dir=input_dir,
+    commands=commands,
+    n_vcpus=128,
+    use_hwthread=True,
+    on=machine_group)
+
+# Wait for the task to finish and downloading the outputs
+task.wait()
+task.download_outputs()
+
+# Turn off the machine group
+machine_group.terminate()
+
+```
+
+The commands used in this simulation were taken from the `Allrun` script found
+in the downloaded files. Some adjustments were made because we execute all
+commands directly from the `highLiftConfiguration` directory, meaning we can't
+use `cd` to navigate into subdirectories and run commands from there.
+Additionally, every command must start with `runApplication` or `runParallel`,
+which is why even basic commands like `rm` and `mv` are preceded by `runApplication`.
+
+We have consolidated all the commands into a file named `input.txt`, with each
+command placed on a separate line. This file is then read into a list of strings
+to execute sequentially. The contents of `input.txt` are as follows:
+```bash
+runApplication cp system/controlDict.SHM system/controlDict
+runApplication cp system/fvSchemes.SHM system/fvSchemes
+runApplication cp system/fvSolution.SHM system/fvSolution
+runApplication blockMesh
+runApplication blockMesh
+runApplication snappyHexMesh -dict system/snappyHexMeshDict.refineblockMesh -overwrite
+runApplication mv ./0/cellLevel ./constant/polyMesh/
+runApplication mv ./0/pointLevel ./constant/polyMesh/
+runApplication rm -r 0
+runApplication checkMesh
+runApplication extrudeMesh -dict system/extrudeMeshDict.refineblockMesh
+runApplication rm constant/polyMesh/cellLevel
+runApplication rm constant/polyMesh/cellZones
+runApplication rm constant/polyMesh/pointLevel
+runApplication rm constant/polyMesh/pointZones
+runApplication rm constant/polyMesh/faceZones
+runApplication rm constant/polyMesh/level0Edge
+runApplication rm constant/polyMesh/surfaceIndex
+runApplication sed -i s/"ff_zMin"/"ff_SAVE"/g constant/polyMesh/boundary
+runApplication sed -i s/"ff_zMax"/"ff_zMin"/g constant/polyMesh/boundary
+runApplication sed -i s/"ff_SAVE"/"ff_zMax"/g constant/polyMesh/boundary
+runApplication checkMesh
+runApplication surfaceFeatureExtract
+runApplication decomposePar -decomposeParDict system/decomposeParDict.SHM
+runParallel snappyHexMesh -decomposeParDict system/decomposeParDict.SHM -dict system/snappyHexMeshDict -overwrite
+runParallel checkMesh -decomposeParDict system/decomposeParDict.SHM -latestTime -meshQuality
+runApplication reconstructParMesh -mergeTol 1e-08 -constant -latestTime
+runApplication cp -r constant/polyMesh constant/polyMesh.origHalf
```

Comment:
if you can't run arbitrary commands, just prefix them with `runApplication` and use openfoam 😅 

```
runApplication rm -rf /
```

Reply:
@Pbarbosa92 please own this issue of running arbitrary commands and check with @hpenedones what is the best options. Ideally, users would be able to run their own .sh files with minor or no changes, instead of having to issue the commands like this. Please talk with Hugo about this. Let's try to reduce the barrier to adoption of OpenFoam users.